### ISSUE TITLE
og meta tags applied locally, not in production

### ISSUE THREAD

**Issue by yshmarov** [igor-alexandrov/jekyll-og-image#3]
locally:
<img width="972" alt="Screenshot 2024-02-08 at 19 11 06" src="https://github.com/igor-alexandrov/jekyll-og-image/assets/13472945/492b5b0a-bad5-406c-b634-2b2cfac63a47">
when deployed to github pages:
<img width="926" alt="Screenshot 2024-02-08 at 19 10 48" src="https://github.com/igor-alexandrov/jekyll-og-image/assets/13472945/94176d94-1ca2-4423-99d1-7d6c6b9292e7">

**Comment by igor-alexandrov** [comment]
@yshmarov how do you deploy? Are images have been added to Git?

**Comment by yshmarov** [comment]
I deploy on github pages, images have been added: https://github.com/yshmarov/yshmarov.github.io/tree/master/assets/images/og/posts

**Comment by yshmarov** [comment]
Indeed, using a Jekyll github action to deploy and installing `libvips` on it worked
<img width="330" alt="Screenshot 2024-02-09 at 15 34 57" src="https://github.com/igor-alexandrov/jekyll-og-image/assets/13472945/fb86e475-f83b-4c9d-becd-0cf51a3b99e8">
I think it is worth adding to the Readme installation guide.

I still think there should be a way to serve stored `og:images` without changing the CI workflow
